go.mod

```diff
@@ -22,7 +22,9 @@ replace (
 	github.com/Azure/go-autorest v11.1.2+incompatible => github.com/Azure/go-autorest/autorest v0.10.0
 	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.1.0
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
+	github.com/microsoft/moc => github.com/microsoft/moc v0.11.0-alpha.6.0.20230602204619-40265f348741 //Sethbe
```

Comment:
Reminder to fix before final commit.
Should be updated onces this PR is completed